ciao-image/datastore/store.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/01org/ciao/clogger/gloginterface"
 	"github.com/01org/ciao/database"
 	"github.com/01org/ciao/openstack/image"
+	"github.com/golang/glog"
 )
```

Comment:
For logging, I think we should start using the  Ciao [clogger](https://github.com/01org/ciao/tree/master/clogger).  You could take the database subproject as an example https://github.com/01org/ciao/tree/master/database